src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar.domain
+
+class Cars(registerCarNumber: Int) {
+
+    val cars: List<Car>
+
+    init {
+        cars = registerCar(registerCarNumber)
+    }
+
+    private fun registerCar(registerCarNumber: Int): List<Car> {
+        return (REGISTER_CAR_START_NUMBER..registerCarNumber).map { Car() }
```

Comment:
아래와 같이 선언하는 방식도 존재합니다! 

```suggestion
        return List(registerCarNumber) { Car() }
```

Reply:
와 코틀린 List Collections에서 이런 부분의 API까지 있는지 몰랐네요!
꿀팁 감사드립니다! 🙇🏻‍♂️